core/sdk-core/src/test/java/software/amazon/awssdk/core/internal/http/response/NullErrorResponseHandler.java

```diff
@@ -25,7 +25,9 @@ public class NullErrorResponseHandler implements HttpResponseHandler<SdkServiceE
     @Override
     public SdkServiceException handle(SdkHttpFullResponse response,
                                       ExecutionAttributes executionAttributes) throws Exception {
-        return SdkServiceException.builder().build();
+        return SdkServiceException.builder()
+            .statusCode(response.statusCode())
```

Comment:
How are we ensuring `SdkHttpFullResponse response` is never null here?

Reply:
We don't, this class is only used for testing.